index.html

```diff
@@ -0,0 +1,276 @@
+          <div
+            class="featured-speakers-1 d-block d-md-flex justify-content-center mb-3 mb-md-0"
+          >
+            <img
+              src="images/ayodeji.png"
+              alt="speaker one"
+              class="mx-auto mb-4 mb-md-0"
+            />
+            <div
+              class="d-flex flex-column justify-content-center align-items-md-start align-items-center"
+            >
+              <h5 class="primary">Sulaimon Ayodeji</h5>
+              <h6 class="secondary-color">
+                Seasoned gospel artist for the past 9 years
+              </h6>
+              <p>
+                Ayodeji has soulful hymns & echo salvation's grace, inspiring
+                hearts with divine melodies. He is gospel artist igniting
+                faith."
+              </p>
+            </div>
+          </div>
+          <div
+            class="featured-speakers-1 d-block d-md-flex justify-content-center mb-3 mb-md-0"
+          >
+            <img
+              src="images/donald.png"
+              alt="speaker one"
+              class="mx-auto mb-4 mb-md-0"
+            />
+            <div
+              class="d-flex flex-column justify-content-center align-items-md-start align-items-center"
+            >
+              <h5 class="primary">Sulaimon Donald</h5>
+              <h6 class="secondary-color">
+                Seasoned gospel artist for the past 9 years
+              </h6>
+              <p>
+                Ayodeji has soulful hymns & echo salvation's grace, inspiring
+                hearts with divine melodies. He is gospel artist igniting
+                faith."
+              </p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section
+      id="partner"
+      class="d-flex flex-column justify-content-center align-items-center accent-color"
+    >
+      <h4 class="partner-header mb-3 mb-md-0 primary accent-color">Partner</h4>
+      <div class="title-line mb-3 mb-md-0"></div>
+      <div
+        class="container d-block d-md-flex justify-content-center align-items-center"
+      >
+        <a href="#" class="accent-color">
+          <h4 class="m-0">RCCG Nigeria</h4>
+        </a>
+      </div>
+    </section>
+    <section id="footer">
+      <div
+        class="container d-block d-md-flex justify-content-center align-items-center"
+      >
+        <div>
+          <a class="navbar-brand" href="index.html"
+            ><img src="images/logo.png" alt="concert logo"
+          /></a>
+        </div>
+        <div>
+          <p class="primary">
+            The Night of Recovery Concert is committed to creating a safe and
+            inclusive environment for all attendees. We encourage responsible
+            behavior and ask everyone to respect each other's journey.
+          </p>
+        </div>
+      </div>
+    </section>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.slim.min.js"></script>
```

Comment:
- [ ] You did great implementing the footer. :+1: :+1:  But looking at the given design the footer image and text is placed side by side both on desktop and mobile, I'll suggest you update your styling for this section as yours tends to stack on top of the text on mobile screens.
- - Your design:
![Screenshot from 2023-05-23 16-43-19](https://github.com/sulaimonaa/music-concert/assets/107632780/ebc30414-6891-4b6c-a622-19e77e21babd)

- Provided Design:
![Screenshot from 2023-05-23 16-43-02](https://github.com/sulaimonaa/music-concert/assets/107632780/5175a1aa-fedc-4fae-a08b-732d019bc89c)